packages/app/src/runner/event-manager.ts

```diff
@@ -361,19 +360,31 @@ export class EventManager {
   }
 
   setup (config) {
-    Cypress = this.Cypress = this.$CypressDriver.create(config)
+    this.ws.emit('watch:test:file', config.spec)
```

Comment:
Do we want to watch the test file during run mode? This sounds like an open mode specific thing.

Reply:
Eg should this be under the check on line 365-368?

Reply:
It likely should but that is outside of the scope of this change and would require additionally testing. Going to leave as it for the PR.